### PR TITLE
ADCM-3115 Add user to audit log of finished actions

### PR DESCRIPTION
**Pull request by winmasta**

### CODE REVIEW

File: python/audit/utils.py [hunk 1]
```diff
@@ -253,25 +252,26 @@ def audit_finish_task(obj, action_display_name: str, status: str) -> None:
...
+        object_name=obj.name,
         object_type=obj_type,
     )
     if status == "success":
         operation_result = AuditLogOperationResult.Success
     else:
         operation_result = AuditLogOperationResult.Fail
 
+    user = None
+    audit_log = AuditLog.objects.filter(audit_object=audit_object).first()
```
Comment (a-alferov) [inline python/audit/utils.py:266]: This request does not guarantee the determination of the correct user. We want the correct user to be the one who started the task.

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/audit/utils.py (hunk 1 above)
Comment (winmasta) [reply python/audit/utils.py]: Refactored

File: python/audit/cases/common.py [hunk 2]
```diff
@@ -83,17 +83,24 @@ def get_obj_name(obj: Model, obj_type: str) -> str:
...
+def get_or_create_audit_obj(
+        object_id: str,
+        object_name: str,
+        object_type: str,
+        action_id: int | None = None,
+) -> AuditObject:
     audit_object = AuditObject.objects.filter(
         object_id=object_id,
         object_type=object_type,
+        action_id=action_id,
```
Comment (a-alferov) [inline python/audit/cases/common.py:95]: Since `action_id` can be `null=True`, there is a chance to get 2 entries for one object

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/audit/cases/common.py (hunk 2 above)
Comment (winmasta) [reply python/audit/cases/common.py]: refactored

**Review by a-alferov — APPROVED** [review]

**Pull request merged**